src/resources/translations/untranslated.ts

```diff
@@ -19472,11 +19447,6 @@ Coordinates plot.</source>
         <source>M3D-C1 3D</source>
         <translation type="unfinished"></translation>
     </message>
-    <message>
-        <location filename="../../plots/Streamline/QvisStreamlinePlotWindow.C" line="543"/>
-        <source>NIMROD</source>
-        <translation type="unfinished"></translation>
-    </message>
     <message>
         <location filename="../../plots/Streamline/QvisStreamlinePlotWindow.C" line="539"/>
```

Comment:
Not related to the removal of NIMROD, but Streamline plot no longer exists, so should probably be removed from here. Same with FTLE operator. These are just what I see in the diff.